Lib/defconQt/featureTextEditor.py

```diff
@@ -17,11 +17,11 @@ def __init__(self, font=None, parent=None):
         self.editor = FeatureTextEditor(self.font.features.text, self)
         self.resize(600, 500)
 
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&Save…", self.save, QKeySequence.Save)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&Save") + "…", self.save, QKeySequence.Save)
```

Comment:
You mean the ellipsis? Sadly there is a need: "pylupdate5 gets confused when strings contain non-ASCII characters". Unless you know a way to give the tool the correct slap on the back of its' head?